### PR TITLE
Fix coordinator state loading

### PR DESCRIPTION
**Pull request by elias-garcia**
### What does this PR does?

This PR loads the `coordinator state` when launching the app and then each 30s.

## Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Respect code style and lint
- [x] Update documentation (if needed)

### CODE REVIEW

File: src/store/global/global.thunks.js [hunk 1]
```diff
@@ -86,6 +87,10 @@ function changeRedirectRoute (redirectRoute) {
  */
 function fetchFiatExchangeRates (symbols) {
```
Comment (AlbertoElias) [inline src/store/global/global.thunks.js:88]: Maybe you don't need the parameter anymore?

File: src/store/transaction/transaction.thunks.js [hunk 2]
```diff
@@ -196,7 +196,9 @@ function fetchAccountBalance () {
  * @returns {void}
  */
 function fetchFees () {
-  return function (dispatch) {
+  return function (dispatch, getState) {
+    const { global: { coordinatorStateTask } } = getState()
+    console.log(coordinatorStateTask)
```
Comment (AlbertoElias) [inline src/store/transaction/transaction.thunks.js:201]: leftover?

**Review by AlbertoElias — CHANGES_REQUESTED** [review]
Couple comments

File: src/store/global/global.thunks.js (hunk 1 above)
Comment (elias-garcia) [reply src/store/global/global.thunks.js:88]: yeah, nice catch!

**Review by AlbertoElias — APPROVED** [review]

**Pull request merged**